client/src/components/ThesesGanttChart/ThesesGanttChart.tsx

```diff
@@ -0,0 +1,5 @@
+const ThesesGanttChart = () => {
+  return <></>
```

Comment:
Can this file be deleted?

Reply:
Right now this is a placeholder file for a future PR, so it will be implemented eventually

Reply:
Ah, I see. 
If this has nothing to do with this PR I would keep it as a local change next time.

Reply:
yeah I agree. Will do that next time